api/src/test/java/org/openmrs/EncounterTest.java

```diff
@@ -1406,75 +1425,31 @@ public void hasDiagnosis_shouldReturnFalseIfEncounterDoesNotHaveDiagnosis() {
 	 */
 	@Test
 	public void getAllConditions_shouldReturnAllConditions() {
-
-		Condition activeCondition = new Condition();
-		activeCondition.setClinicalStatus(ConditionClinicalStatus.ACTIVE);
-		CodedOrFreeText freeText = new CodedOrFreeText();
-		freeText.setNonCoded("Free text");
-		activeCondition.setCondition(freeText);
-		encounter.addCondition(activeCondition);
-
-		Condition voidedCondition = new Condition();
-		voidedCondition.setVoided(true);
-		voidedCondition.setClinicalStatus(ConditionClinicalStatus.HISTORY_OF);
-		CodedOrFreeText freeText01 = new CodedOrFreeText();
-		freeText01.setNonCoded("Free text");
-		voidedCondition.setCondition(freeText01);
-		encounter.addCondition(voidedCondition);
-
-		assertEquals(2, encounter.getAllConditions(true).size());
+		
+		assertEquals(2, encounter.getConditions(true).size());
 	}
 
 	/**
 	 * @see Encounter#getConditions()
 	 */
 	@Test
 	public void getConditions_shouldReturnActiveConditions() {
```

Comment:
`getConditions_shouldReturnNonVoidedConditionsByDefault`

Reply:
It's already been done in one of the older commits.

Reply:
That the conditions are voided and not deleted.

Reply:
Done.